### DISCUSSION TITLE
Modification priority

### DISCUSSION THREAD

**Discussion by emmaahrman** [vdemichev/DiaNN#553] [Q&A]
Hello,
I have used diaNN 1.8 to quantify proline hydroxylation as variable modification, but had Met(ox) enabled during search. Does this setting prioritise met(ox) as fixed mod over variable pro(ox)? Both of them were labelled UniMod:35, and there was no peptide that contained only Pro modified that didn't contain Met modified.
Thank you.

**Comment by vdemichev** [comment]
M(ox) is variable, should be treated exactly the same as all other variable mods. That is they are all equally prioritised. 
Try use --strip-unknown-mods, if using in silico prediction? Also --original-mods?
